### PR TITLE
feat(issues): Move event context out of tags

### PR DESCRIPTION
**Pull request by scttcper**
Most of the style changes were to get the divider to fully draw the border between the two components

before
image

after - without screenshot
image

after - with screenshot
Screen Shot 2022-07-20 at 2 38 15 PM

Fixes https://getsentry.atlassian.net/browse/WOR-2044

### CODE REVIEW

**Comment by priscilawebdev** [comment]
do we also want a full line between the context and tags on smaller devices?

image

**Comment by priscilawebdev** [comment]
should we hide the divider if there is no context? 

image

**Comment by priscilawebdev** [comment]
image

**Comment by scttcper** [comment]
@priscilawebdev i think i fixed most of the mobile issues, thanks for checking them. This component is tricky with how many combinations there are

**Comment by priscilawebdev** [comment]
Thanks @scttcper 🙏

I can still see the divider when there is no context

image

but other than that, it looks good

**Comment by scttcper** [comment]
@priscilawebdev where are you able to reproduce this? It shouldn't render the divider if there isn't any context https://github.com/getsentry/sentry/blob/scttcper/break-out-context/static/app/components/events/eventTagsAndScreenshot/tags.tsx#L27-L34

**Comment by priscilawebdev** [comment]
@scttcper I will send you the event 😉 I think the issue is happening because of the` event.user` which is not empty and the following logic makes it return null 

https://github.com/getsentry/sentry/blob/49cbda5a6b8f3b78d185f614c85428fb89701e8c/static/app/components/events/contextSummary/index.tsx#L80

**Comment by scttcper** [comment]
@priscilawebdev i checked for the bailout condition a little higher and should prevent the rendering of the line

File: static/app/components/events/eventTagsAndScreenshot/index.tsx [hunk 1]
```diff
@@ -73,17 +67,37 @@ function EventTagsAndScreenshots({
 
 export default EventTagsAndScreenshots;
 
+/**
+ * Used to adjust padding based on which 3 elements are shown
+ * - screenshot
+ * - context
+ * - tags
```
Comment (priscilawebdev) [inline static/app/components/events/eventTagsAndScreenshot/index.tsx:74]: nice 🙂

**Comment by priscilawebdev** [comment]
@scttcper yes it solves the divider issue, but should we also update the padding to the titles are aligned?

image

**Comment by scttcper** [comment]
@priscilawebdev oh no.. i've moved the check a bit higher in the tree to check for that

**Comment by priscilawebdev** [comment]
Almost there :)

I think we should only add some left padding now and it shall be good

image

**Comment by scttcper** [comment]
@priscilawebdev thanks, i switched to doing the styles in a way i hope is easier to read

**Comment by priscilawebdev** [comment]
changes look good, but I found this other issue :/

**Before:**

image

**After:**
image

File: static/app/components/events/eventTagsAndScreenshot/index.tsx [hunk 2]
```diff
@@ -40,88 +43,110 @@ function EventTagsAndScreenshots({
...
+        </ScreenshotWrapper>
       )}
+      {showScreenshot && showTags && <VerticalDivider />}
+      <TagWrapper hasEventContext={hasEventContext}>
+        {hasEventContext && (
+          <Fragment>
+            <TagsHighlightWrapper showScreenshot={showScreenshot}>
+              <TagsHighlight event={event} />
+            </TagsHighlightWrapper>
+            <Divider />
```
Comment (priscilawebdev) [inline static/app/components/events/eventTagsAndScreenshot/index.tsx:70]: what happens if `hasEventContext` equals true and `showTags` false? are we displaying the divider? does it look good?

Comment (scttcper) [reply static/app/components/events/eventTagsAndScreenshot/index.tsx]: i've blocked this now

**Comment by scttcper** [comment]
I think i've fixed the overflow issues and some issues with the screenshot loading size 
image

**Comment by priscilawebdev** [comment]
can confirm that the last changes solved the issues. Thank you 🙏

I think I found another minor thing. should we add some padding-left to the contexts section? 

**Before:**
image


**After:**

image

**Review by priscilawebdev — APPROVED** [review]
left 1 last suggestion but other than that it looks good to me

**Pull request merged**